recipes-graphics/spark/spark_git.bb

```diff
@@ -0,0 +1,100 @@
+include pxcore.inc
+
+DEPENDS_append = " curl freetype util-linux libjpeg-turbo libpng pxcore-libnode giflib sqlite3 wpeframework-tools-native wpeframework wpeframework-clientlibraries"
```

Comment:
Do you still need the wpeframework-tools-native here? Perhaps it's better to include the wpeframework.inc file

Reply:
yes, new config generator has adding this dependency with all package wherever we need WPEFramworkConfig.cmake. Hence we need this for spark as well. Do you mean to create new wpeframework.inc for all these thunder stuffs ?